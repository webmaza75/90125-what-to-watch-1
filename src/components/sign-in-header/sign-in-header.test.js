import React from 'react';
import renderer from 'react-test-renderer';

import SignInHeader from './sign-in-header.jsx';

it(`SignInHeader correctly renders`, () => {
  const signInHeader = renderer
    .create(<SignInHeader />)
    .toJSON();

  expect(signInHeader).toMatchSnapshot();
});
