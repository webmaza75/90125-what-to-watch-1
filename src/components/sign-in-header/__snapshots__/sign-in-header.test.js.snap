// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInHeader correctly renders 1`] = `
<header
  className="page-header user-page__head"
>
  <div
    className="logo"
  >
    <a
      className="logo__link "
      href="main.html"
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
  <h1
    className="page-title user-page__title"
  >
    Sign in
  </h1>
</header>
`;
